src/Humanizer.Tests.Shared/CollectionHumanizeTests.cs

```diff
@@ -87,5 +88,58 @@ public void HumanizeUsesObjectFormatterWhenSeparatorIsProvided()
             var humanized = _testCollection.Humanize(sc => string.Format("SomeObject #{0} - {1}", sc.SomeInt, sc.SomeString), "or");
             Assert.Equal("SomeObject #1 - One, SomeObject #2 - Two, or SomeObject #3 - Three", humanized);
         }
+
+        [Fact]
+        public void HumanizeHandlesNullItemsWithoutAnException()
+        {
+            new object[] { null, null }.Humanize();
```

Comment:
Sure, but I find `DoesNotThrow` redundant and not in a nice way. The whole premise of its existence flies in the face of how the language itself works.

It's on track to be removed from xunit, and I have to say I agree with the sentiments there:  https://github.com/xunit/xunit/issues/188

Other projects like dotnet/roslyn have been removing all usages of `DoesNotThrow`. I'd hate to perpetuate it. Do I have to?